sdk/storage/Azure.Storage.Blobs/src/Models/CopyFromUriOperation.cs

```diff
@@ -134,7 +138,7 @@ internal class CopyFromUriOperation : Operation<long>
         /// notifications that the operation should be cancelled.
         /// </param>
         /// <returns>The <see cref="Response"/> with the status update.</returns>
-        public async override ValueTask<Response> UpdateStatusAsync(
+        public override async ValueTask<Response> UpdateStatusAsync(
             CancellationToken cancellationToken = default) =>
             await UpdateStatusAsync(true, cancellationToken).ConfigureAwait(false);
 
```

Comment:
What are your thoughts on trying to expose the `enum CopyStatus { Pending, Success, Aborted, Failed }` somewhere on this type?  I was kind of hoping we could use it as the primary interface instead of `HasCompleted` and `HasValue`.  I'm fine doing this in a future release if we think there's a reasonable non-breaking path forward.  I'm confident we can add `Abort`/`AbortAsync` in a future release so don't want to bother now.